src/app/shared/post-it-collections/import-button/post-it-collection-import-button.component.html

```diff
@@ -0,0 +1,24 @@
+<button type="button"
+        class="btn btn-outline-{{ disabled ? 'secondary' : 'primary' }}"
```

Comment:
Why do we use `primary` color here? It does not fit to our design at all. It should be `success`.

The other thing is that the button should probably not be shown when the user is not able to click it.